dev/template_dev/template_dev.py

```
@@ -113,34 +116,47 @@
 
 
-def generate(product, template, parameters: dict[str, str] | None = None) -> dict[str, bytes | str]:
+def generate(product, template, parameters: dict[str, str] | None = None) -> bytes:
     if parameters is None:
         parameters = {}
 
-    try:
-        env = jinja2.Environment(autoescape=False)
-        tmpl = env.from_string(template)
-        product["current_date"] = datetime.datetime.now().strftime("%Y-%m-%d")
+    env = jinja2.Environment(autoescape=False)
```

Ruff (0.15.13)

[error] Using jinja2 templates with `autoescape=False` is dangerous and can lead to XSS. Ensure `autoescape=True` or use the `select_autoescape` function.

(S701)